editor/components/block-list/style.scss

```diff
@@ -1,3 +1,71 @@
+.components-draggable__clone {
+	.editor-block-list__block-draggable {
+		background: white;
+		box-shadow: $shadow-popover;
+
+		@include break-small {
+			left: 0;
+			right: 0;
+		}
+	}
+
+	// Hide the Block UI when dragging the block
+	// This ensures the page scroll properly (no sticky elements)
+	.editor-block-contextual-toolbar,
+	.editor-block-mover,
+	.editor-block-settings-menu {
+		// I think important is fine here to avoid over complexing the selector
+		display: none !important;
+	}
+}
+
+.editor-block-list__block-draggable {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	z-index: z-index( '.editor-block-list__block-drag-inset' );
+
+	> .editor-block-list__block-draggable-inner {
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		background-color: $light-gray-200;
+		visibility: hidden;
+
+		@include break-small {
+			margin: 0 48px;
+		}
+	}
+
+	&.is-visible > .editor-block-list__block-draggable-inner {
+		visibility: visible;
+		z-index: z-index( '.editor-block-list__block-drag-inset.is-visible > .editor-block-list__block-draggable-inner' );
+	}
+
+	@include break-small {
+		left: -$block-side-ui-padding;
+		right: -$block-side-ui-padding;
+	}
+
+	cursor: move;/* Fallback for IE/Edge < 14 */
+	cursor: grab;
+}
+
+.editor-block-list__block {
+	&.is-hidden *,
+	&.is-hidden > * {
+		visibility: hidden;
+	}
+
+	.editor-block-list__block-edit .reusable-block-edit-panel * {
+		z-index: z-index( '.editor-block-list__block-edit .reusable-block-edit-panel *' );
```

Comment:
What's this for? Does it need to be applied as wildcard to all its children? Or is the container enough?

Reply:
I don't know :) cc @chriskmnds

Reply:
I've answered this above: https://github.com/WordPress/gutenberg/pull/4115/files/51c1cefc4081f4965d1db68fe7e6dd6f02ae7d8e#r178302159 Not sure why I decided to apply it on all its immediate children, but might be due to multiple controls being blocked in the panel.

Reply:
I'd like to see if we can simplify this to eliminate the wildcard selector.